kafkatest/go_verifiable_consumer/go_verifiable_consumer.go

```diff
@@ -0,0 +1,435 @@
+/**
```

Comment:
These are smaller than the Java versions, but not by much. Let's discuss ways we can reduce the overhead of implementing these while retaining the important functionality for tests -- I assumed the Java versions were particularly verbose because of Java.